peps/pep-0536.rst

```diff
@@ -3,12 +3,16 @@ Title: Final Grammar for Literal String Interpolation
 Version: $Revision$
 Last-Modified: $Date$
 Author: Philipp Angerer <phil.angerer@gmail.com>
-Status: Deferred
+Status: Withdrawn
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 11-Dec-2016
 Python-Version: 3.7
-Post-History: 12-Dec-2016
+Post-History: `18-Aug-2016 <https://mail.python.org/archives/list/python-ideas@python.org/thread/FOYKXOFWEINPVQSK2XGEHKXSTEVO5WWA/
+>`__
+              `23-Dec-2016 <https://mail.python.org/archives/list/python-ideas@python.org/thread/YKKEA5NIMMKHZTMRE5UFHST4WQ4NN3XJ/>`__
+              `15-Mar-2019 <https://mail.python.org/archives/list/python-dev@python.org/thread/N43O4KNLZW4U7YZC4NVPCETZIVRDUVU2/>`__
```

Comment:
We're missing commas here:

(```'Post-History' must be '`DD-mmm-YYYY <Thread URL>`__, ...'```)

```suggestion
Post-History: `18-Aug-2016 <https://mail.python.org/archives/list/python-ideas@python.org/thread/FOYKXOFWEINPVQSK2XGEHKXSTEVO5WWA/
>`__,
              `23-Dec-2016 <https://mail.python.org/archives/list/python-ideas@python.org/thread/YKKEA5NIMMKHZTMRE5UFHST4WQ4NN3XJ/>`__,
              `15-Mar-2019 <https://mail.python.org/archives/list/python-dev@python.org/thread/N43O4KNLZW4U7YZC4NVPCETZIVRDUVU2/>`__
```

Reply:
aah, I see!

Reply:
ah, I see, there was a superfluous line break, and the validator didn’t help with that. Fixed both